action.yml

```diff
@@ -83,15 +91,58 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Log the next step action
+    run: echo ▷ Figuring out the environment Python version...
+    shell: bash
+  - name: Compute job Python version
+    id: compute-origin-python-version
+    run: |
+      # Compute job Python version
+      import sys
+
+      # Input from GHA
+      origin_python_version = '${{ inputs.origin-python-version }}'
+      old_python_version = '${{ inputs.python-version }}'
+      ansible_core_version = '${{ inputs.ansible-core-version }}'
+
+      # Handle case that deprecated Python version parameter is provided
+      if old_python_version:
+          # Check whether the default of origin-python-version was changed
+          if origin_python_version != 'auto':
+              sys.exit(
+                  '::error ::`python-version` and `origin-python-version` cannot'
+                  ' both be supplied. Use only `origin-python-version` and not'
+                  ' its deprecated alias `python-version`.'
+              )
+
+          origin_python_version = old_python_version
+
+      if origin_python_version == 'auto':
+          # Determine maximal Python version supported by the
+          # given ansible-core version.
+          if ansible_core_version in ('stable-2.9', ):
+              origin_python_version = '3.8'
+          elif ansible_core_version in ('stable-2.10', 'stable-2.11'):
+              origin_python_version = '3.9'
+          elif ansible_core_version in ('stable-2.12', 'stable-2.13'):
+              origin_python_version = '3.10'
+          else:
+              # For any non-stable branch, we assume the latest version
+              # supported by 'devel':
+              origin_python_version = '3.10'
+
+      # Set computed origin-python-version
+      print(f'::set-output name=origin-python-version::{origin_python_version}')
```

Comment:
How about also exposing this via `outputs` above?

Reply:
Related: https://github.com/ansible-community/ansible-test-gh-action/pull/19/files#r945835544

Reply:
Implemented in 108243573eea9e19722c75a30e978472ec8619d9.